content/docs/command-reference/exp/run.md

```diff
@@ -35,13 +35,20 @@ See the [Options](#options) section for the differences.
 
 </admon>
 
-Use the `--set-param` (`-S`) option as a shortcut to change
-<abbr>parameter</abbr> values [on-the-fly] before running the experiment.
-
 It's possible to [queue experiments] for later execution with the `--queue`
 flag. Queued experiments can be run with `dvc queue start` and managed with
 other `dvc queue` commands.
 
+Use the `--set-param` (`-S`) option as a shortcut to change
+<abbr>parameter</abbr> values [on-the-fly] before running the experiment. DVC
+supports Hydra's [Basic Override syntax] for parsing values passed to
+`--set-param`. In addition,
+[Choice](https://hydra.cc/docs/advanced/override_grammar/extended/#choice-sweep)
+and/or
+[Range](https://hydra.cc/docs/advanced/override_grammar/extended/#range-sweep)
+syntax can be combined with `--queue` in order to add multiple experiments to
+the queue.
```

Comment:
Not a blocker: I see this is now repeating what is below under options. If it's not saving you from writing anything in the options section, I think you could drop it here. Sorry, I know this is basically the opposite of what I said previously, but I was hoping putting text here would remove the need for it under the options.